webapp/components/test/wpt-results.html

```diff
@@ -64,82 +64,6 @@
             });
           });
         });
-
-        suite('computeDifferences(before, after)', () => {
```

Comment:
Why is this bunch of tests deleted? This isn't the component this PR is changing.

Reply:
Bad merge. reverted.